tests/test_examples.py

```diff
@@ -22,6 +22,11 @@
     'capture_recapture/cjs.py --num-steps=1 -m 3',
     'capture_recapture/cjs.py --num-steps=1 -m 4',
     'capture_recapture/cjs.py --num-steps=1 -m 5',
+    'capture_recapture/cjs.py --num-steps=1 -m 1 --tmc --tmc-num-samples=2',
```

Comment:
why use tmc in scenarios where i can compute the exact marginal likelihood?

Reply:
You shouldn't, I just think it's neat that TMC makes Monte Carlo gradient estimation feasible in these models.

Reply:
ok but anywhere this is true (hmm, cjs) we should make a comment to the effect that is illustrative only